### PR TITLE
Update gui readme

### PR DESCRIPTION
**Pull request by blacksponge**
## Purpose

Update the readme to remove outdated information and make it more like other components readme.

## Context

\-

## Changes

* Remove some contribution text
* Remove sécurity section because that has not be done for so long
* Remove duplicated acknowledgment to angular cli
* Remove release instructions
* Reorder some sections
* Update the build badge to use github action

## How to test this PR

\-

### CODE REVIEW

**Review by marc-vanderwal — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -17,95 +15,72 @@ installation] document.
...
+All assets are available in `src/assets` folder. It's split by concern, the
+style in the `css`, images are in the ``images`` folder, etc.
+
+All translation files are in the `src/locale` folder
+
+The environment defaults of the application are in the `src/environments`
+folder. `environment.ts` is use for development purpose, the
+`environment.prod.ts` for production, and the `environment.test.ts` for testing.
 
 In order to contribute
```
[the remark this answers is not in the document]
Comment (matsduf) [reply README.md:58]: No space before ":" in English.

File: README.md [hunk 2]
```diff
@@ -17,95 +15,72 @@ installation] document.
...
 configuration] document.
 
+### Documentation
+
+There is a [public documentation]. Some more specific documents can be found in
+the [docs directory](docs/).
 
 ### Contribution
 
 This project was generate with Angular-CLI 1.6.8 and then updating.
```
Comment (matsduf) [inline README.md:37]: What does the sentence mean?

File: README.md [hunk 3]
```diff
@@ -17,95 +15,72 @@ installation] document.
...
-
-The configurations files of the application are in the ``src/environments`` folder.
-``environment.ts`` is use for development purpose, and the ``environment.prod.ts`` for production.
+The source code of the application is available in `./src/app` folder.
+
+The `components` folder is composed of subfolders that represent the main
+components. Each folder of component contain three files :
+
+  * `*.component.css` : The component's style
+  * `*.component.html` : The component's html
```
[the remark this answers is not in the document]
Comment (matsduf) [reply README.md:46]: "The component's style configuration"?
"The component's HTML code"?

File: README.md [hunk 4]
```diff
@@ -17,95 +15,72 @@ installation] document.
...
+  * `*.component.ts` : The component's typescript
```
[the remark this answers is not in the document]
Comment (matsduf) [reply README.md:47]: "The component's TypeScript code"?

File: README.md [hunk 5]
```diff
@@ -17,95 +15,72 @@ installation] document.
...
+
+All assets are available in `src/assets` folder. It's split by concern, the
```
[the remark this answers is not in the document]
Comment (matsduf) [reply README.md:49]: "It is split by concern; the style configuration is in the `css`, images are in the `images` folder, etc."

**Comment by blacksponge** [comment]
I hope to have addressed all comments.

File: README.md (hunk 2 above)
Comment (blacksponge) [reply README.md]: It is unclear to me as well, I shortened it to "This project was generated with Angular-CLI 1.6.8.", even though I don't know how relevant this information is.

File: README.md [hunk 6]
```diff
@@ -17,95 +15,72 @@ installation] document.
...
 
 In order to contribute
+
 * Install [Nodejs](https://nodejs.org)
 * Fork the Zonemaster GUI repository on Github into your own user on Github.
 * Clone your fork to your working environment.
 * Go to the folder and install project dependencies with `npm install`
-* Update environment files in `/src/environments/` to add a backend api endpoint (or leave the default)
+* Update configuration files in `src/assets/app.config.json` to add a backend
+  api endpoint (or leave the default)
```
[the remark this answers is not in the document]
Comment (blacksponge) [reply README.md]: done.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (blacksponge) [reply README.md]: done (without the space before the colon).

File: README.md [hunk 7]
```diff
@@ -17,95 +15,72 @@ installation] document.
...
+  * `*.component.html` : The component's html
+  * `*.component.ts` : The component's typescript
+
+All assets are available in `src/assets` folder. It's split by concern, the
+style in the `css`, images are in the ``images`` folder, etc.
+
+All translation files are in the `src/locale` folder
+
+The environment defaults of the application are in the `src/environments`
+folder. `environment.ts` is use for development purpose, the
```
[the remark this answers is not in the document]
Comment (blacksponge) [reply README.md]: done.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (blacksponge) [reply README.md]: done

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (blacksponge) [reply README.md]: done

File: README.md [hunk 8]
```diff
@@ -17,95 +15,72 @@ installation] document.
...
 
+### Documentation
+
+There is a [public documentation]. Some more specific documents can be found in
+the [docs directory](docs/).
 
 ### Contribution
 
 This project was generate with Angular-CLI 1.6.8 and then updating.
 It use the Angular 2+ framework with all its tools.
```
[the remark this answers is not in the document]
Comment (blacksponge) [reply README.md]: done

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (blacksponge) [reply README.md]: I went for the second proposal

File: README.md [hunk 9]
```diff
@@ -17,95 +15,72 @@ installation] document.
...
+components. Each folder of component contain three files :
+
+  * `*.component.css` : The component's style
+  * `*.component.html` : The component's html
+  * `*.component.ts` : The component's typescript
+
+All assets are available in `src/assets` folder. It's split by concern, the
+style in the `css`, images are in the ``images`` folder, etc.
+
+All translation files are in the `src/locale` folder
```
[the remark this answers is not in the document]
Comment (blacksponge) [reply README.md]: done

**Pull request merged**